api/data_explorer/controllers/export_url_controller.py

```diff
@@ -9,14 +9,18 @@
 import time
 import urllib
 
+from collections import OrderedDict
```

Comment:
Export with no cohort doesn't work

Reply:
Fixed, now just exports the full samples table (no sample set).